Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Threat Model
+============
+
+In this section, we present a threat model for Cilium. This threat model
+allows interested parties to understand:
+
+-  security-specific implications of Cilium’s architecture
+-  controls that are in place to secure data flowing through Cilium’s various components
+-  recommended controls for running Cilium in a production environment
+
+Scope and Prerequisites
+-----------------------
+
+This threat model considers the possible attacks that could affect an
+up-to-date version of Cilium running in a production environment; it
+will be refreshed when there are significant changes to Cilium’s
+architecture or security posture.
+
+This model does not consider supply-chain attacks, such as attacks where
+a malicious contributor is able to intentionally inject vulnerable code
+into Cilium. For users who are concerned about supply-chain attacks,
+Cilium’s `recent security audit`_ assessed Cilium’s supply chain controls
+against `the SLSA framework`_.
+
+In order to understand the following threat model, readers will need
+familiarity with basic Kubernetes concepts, as well as a high-level
+understanding of Cilium’s `architecture and components`_.
+
+.. _recent security audit: https://github.com/cilium/cilium.io/blob/main/Security-Reports/CiliumSecurityAudit2022.pdf
+.. _the SLSA framework:  https://slsa.dev/
+.. _architecture and components: https://docs.cilium.io/en/stable/overview/component-overview/
+
+Methodology
+-----------
+
+In this threat model, we will consider eight different types of threat
+actor, placed at different parts of a typical deployment stack. We will
+primarily use Kubernetes as an example but the threat model remains
+accurate if deployed with other orchestration systems, or when running
+Cilium outside of Kubernetes. The attackers will have different levels
+of initial privileges, giving us a broad overview of the security
+guarantees that Cilium can provide depending on the nature of the threat
+and the extent of a previous compromise.
+
+For each threat actor, we assess the likely attacks using `the STRIDE
+methodology`_. Where one attack type in the STRIDE set can lead to others
+(for example, tampering leading to denial of service), we have described the
+attack path under the most impactful attack type. For the potential attacks
+that we identify, we recommend controls that can be used to reduce the
+risk of the identified attacks compromising a cluster. Applying the
+recommended controls is strongly advised in order to run Cilium securely
+in production.
+
+.. _the STRIDE methodology: https://en.wikipedia.org/wiki/STRIDE_(security)
+
+Reference Architecture
+----------------------
+
+For ease of understanding, we are going to consider a single Kubernetes
```

Comment:
In this case we can use the imperative, to instruct the reader about how to think about the problem:
```suggestion
For ease of understanding, consider a single Kubernetes
```